episodes/excthreesteps.Rmd

```diff
@@ -0,0 +1,455 @@
+---
+title: 'Demons Image Registration'
+teaching: 10
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How to understand and visualise three image in one pane for demons image registration algorithm?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Understanding Demons Image Registration code for 3 images viewing pane
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+# Demons Image Registration Algorithm with Multi-Pane Display
+
+This document explains the implementation of a 2D image registration algorithm using the Demons algorithm. The implementation includes visualisation updates within a single window containing three panes for easier comparison of source, target, and warped images.
+
+## 1. Importing Libraries
+
+First, we import the necessary libraries for image processing, transformation, and visualisation. The `utils3` module is another Python file containing utility functions used for tasks such as displaying images and handling deformed and updated fields.

```

Comment:
Can you please create a path `src/mirc/utils/` under the main path of the repo to put `utils3.py` and `demonsReg.py` and adding scripts for exercises as shown below:
```
Medical-Image-Registration-Short-Course
├── ...
├── README.md
├── src
│   └── mirc
│       └── utils
│              └── utils3.py
│              └── demonsReg.py
│       └── exercises
│              └── demons.py
```